src/types.ts

```diff
@@ -6,12 +6,8 @@ import { IntegrationInstanceConfig } from '@jupiterone/integration-sdk-core';
  */
 export interface IntegrationConfig extends IntegrationInstanceConfig {
   /**
-   * The provider API client ID used to authenticate requests.
+   * The provider API key used to authenticate requests.
    */
-  clientId: string;
-
-  /**
-   * The provider API client secret used to authenticate requests.
-   */
-  clientSecret: string;
+  apiKey: string;
+  numRequests: string;
```

Comment:
It will be important to document the meaning and impact of changing this variable. We are going to make it visible in the UI as well (when we deploy it), so this will aid writing the help text for the field.

```suggestion
  /**
   * The number of recent requests to ingest.
   *
   * Requests are Record entities. The collection size will grow indefinitely. This allows for limiting the number
   * of recently created/changed requests to ingest whenever the integration executes. This should be set to
   * a value that correlates to the execution interval of the integration and the expected number of requests
   * that would have been created/changed between executions.
   */
  numRequests: string;
```